packages/lib/src/vis/shared/AxisSystem.module.css

```diff
@@ -52,20 +40,8 @@
   text-anchor: middle;
 }
 
-.title {
```

Comment:
The title is now rendered directly under `.visCanvas`, so I've moved the style to `VisCanvas.module.css`.